tests/lib/rules/class-methods-use-this.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @fileoverview Tests for class-methods-use-this rule.
+ * @author Patrick Williams
+ * @copyright 2016 Patrick Williams. All rights reserved.
```

Comment:
c/p error on \@author? And the copyright is definitely wrong, that should say jQuery foundation